storybook/storybook-react/src/SkipLink/SkipLink.stories.tsx

```diff
@@ -0,0 +1,77 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import { Grid, Paragraph, Screen, SkipLink } from '@amsterdam/design-system-react'
+import { Meta, StoryObj } from '@storybook/react'
+
+const meta = {
+  title: 'Navigation/Skip Link',
+  component: SkipLink,
+  args: {
+    children: 'Direct naar inhoud',
+    href: '#',
+  },
+  argTypes: {
+    style: {
+      table: {
+        disable: true,
+      },
+    },
+  },
+  decorators: [
+    (Story) => (
+      <Screen>
+        <Grid>
+          <Grid.Cell span="all">
+            <Story />
+          </Grid.Cell>
+        </Grid>
+      </Screen>
+    ),
+  ],
+} satisfies Meta<typeof SkipLink>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Default: Story = {
+  args: {
+    style: {
+      clip: 'auto',
+      clipPath: 'none',
+      height: 'auto',
+      overflow: 'auto',
+      position: 'static',
+      whiteSpace: 'normal',
+      width: 'auto',
+    },
+  },
+}
+
+export const OnFocus: Story = {
+  decorators: [
+    (Story) => (
+      <>
+        <Paragraph size="small" style={{ marginBottom: '2rem' }}>
```

Comment:
I have been using `.amsterdam-docs-paragraph` for this. Which of the two would you pick to use everywhere?

Reply:
I think using a `Paragraph` makes more sense. The styling is exactly the same for all stories, and we only have to maintain that styling in one place.